### DISCUSSION TITLE
Screen Section does not compile

### DISCUSSION THREAD

**Discussion by asusjtc** [opensourcecobol/opensourcecobol4j#44] [Q&A]
Good afternoon.
Can you tell me what should I do to avoid the compilation error in a Cobol program with screen section ?
The error is : cannot find symbol s_10
          cob_screen_accept (s_10, null, null);
================================
The partial code is:
_      SCREEN SECTION.
       01  DATA-ENTRY-SCREEN.
         05  VALUE "DATA ENTRY SCREEN" BLANK SCREEN     LINE 1 COL 35.
         05  VALUE "ID #"                               LINE 3 COL 10.
         05   ID-INPUT                                  LINE 3 COL 25
                    PIC  X(4)     TO ID-IN-WS.
         05  VALUE "NAME"                               LINE 5 COL 10.
         05  NAME-INPUT                                 LINE 5 COL 25
                    PIC X(20)     TO NAME-IN-WS.
         05  VALUE "C - TO CONTINUE"                    LINE 11 COL 30.
         05  VALUE "Q - TO QUIT"                        LINE 12 COL 30.
         05  VALUE "ENTER RESPONSE"                     LINE 14 COL 30.
         05  RESPONSE-INPUT                             LINE 14 COL 45
                    PIC X         TO RESPONSE-IN-WS.
       PROCEDURE DIVISION.
       Inicio.
      * Mostramos la pantalla inicial y aceptamos la variable.
      	   	
           display DATA-ENTRY-SCREEN.
          accept DATA-ENTRY-SCREEN.
      	   	
           stop run.
++++++++++++++++++++++
Any help should be appreciated._

**Comment by yutaro-sakamoto** [comment]
Sorry, SCREEN SECTION has not been implemented.

**Reply by asusjtc** [reply]
Dear Yutaro-Sakamoto :
I think that Cobol Screen-section could be implemented as java Jframe , and I will try to do a manual conversion of simple samples.
I will do the work in plain java 11, but working partial time.
After that I will contact you again to show how I did that;  and ask you if you consider possible  to incorporate  into opensourcecogol4j.
Many thanks.
